parser-typechecker/src/Unison/FileParsers.hs

```diff
@@ -198,19 +197,15 @@ synthesizeFile ambient tl fqnsByShortName uf term = do
   applyTdnrDecisions
     :: [Context.InfoNote v Ann]
     -> Term v
-    -> Result' v (Term v)
```

Comment:
The implementation always succeeds with `pure` and never fails, so this function is actually pure.